### PR TITLE
Session 2

### PR DESCRIPTION
**Pull request by sofmeireles**
Merge after #1 

### Added
- methods new, create, show, edit, update to BooksController
- BooksHelper methods
- Services for book creation (CreateBook) and update (UpdateBook)
- Views for edit, new, show (and _form partial)
- Specs for books requests, create and update services
- flash messages on application.html 
- locals for errors and notices (en only)

### Changed
- index view to be more pretty :) and added the new book and edit book buttons
- typo in "success"

### CODE REVIEW

**Comment by pjpires** [comment]
Please, let me know when you merge #1, so I can review this one with only the more recent changes you applied from session 2. 

A nice trick for the next time is to open a PR to your branch from PR #1 instead of main, that way we would be able to see just the new changes isolated from your other commits. Plus, when you merge the first PR into main github will automatically change the base branch on the second PR to main, which is what we want to do most of the times. 😄

**Comment by sofmeireles** [comment]
> Please, let me know when you merge #1, so I can review this one with only the more recent changes you applied from session 2.
> 
> A nice trick for the next time is to open a PR to your branch from PR #1 instead of main, that way we would be able to see just the new changes isolated from your other commits. Plus, when you merge the first PR into main github will automatically change the base branch on the second PR to main, which is what we want to do most of the times. 😄

Oh, I did not know that! Noted, that's pretty cool :) 
Also, #1 is merged!

File: config/locales/en.yml [hunk 1]
```diff
@@ -31,3 +31,18 @@
 
 en:
   hello: "Hello world"
+  activerecord:
```
Comment (pjpires) [inline config/locales/en.yml:34]: Well done! 👌

**Review by pjpires — APPROVED** [review]
Everything looks good to me! 👍 You still have a few inconsistencies in the indentation and spacing here and there, but that's not a big issue. 😄

**Pull request merged**